.github/workflows/tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [12, 14]
```

Comment:
If we're dropping node 12 for the V3, we'll have to update the readme & everywhere there is a reference to node 12 (thinking about the package.json engine versions)

Reply:
It's just the tests, Node 12 is EOL, our docs have always said we only support LTS node releases. I'm not sure if we really need to update the readme and all that jazz for basically *checks notes* 39 days of v3 support left lol.

Reply:
(I only gutted the node 12 tests because they were failing to even start, GH actions didn't even want to install it)

Reply:
If that's by for support to manage it for the next 40days we can merge and release then

Reply:
The readme was saying we support 10 > xx < 14, hence the comment, but if you're fine with it, that's good for me too :slightly_smiling_face: